### PR TITLE
Update respond-file-alerts.md

### PR DESCRIPTION
**Pull request by illfated**
**Changes proposed:**

- Whitespace standardization and normalization
- MarkDown table extended to simplify future editing
- Use MD H4 instead of **bold** style paragraph headings

**Whitespace changes:**

- remove redundant end-of-line whitespace
- whitespace indents: use 3 instead of 4 blank spaces
- add MD indent marker (`> `) compatibility spacing

**Ticket / PR reference or closure:**

Ref. my own comment in PR #8726

### CODE REVIEW

**Review by illfated — COMMENTED** [review]
To skip the trivial whitespace changes (shorten the source page review length) and view the more effective changes in this PR, use the **Hide whitespace changes** feature:
- https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8787/files?diff=unified&w=1

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
Thanks. I guess I missed that one.

**Comment by illfated** [comment]
I did not think much of it at first, but now, using the **Rich Diff view** (https://github.com/MicrosoftDocs/windows-itpro-docs/pull/8787/files?short_path=d0d04ac#diff-d0d04acb98d10029b8ed98c7bffe4302a644fe5f84343094c5b549cd7064b595), I notice that the H4 headings seem to be more contrasted against the white background compared to the previous \*\***bold**\*\* style headings.

**Comment by e0i** [comment]
cc @JohanFreelancer9 for copy editing review. Thanks.

**Comment by illfated** [comment]
Feel free to review the expanded view of the file, if you like. I can always add changes, even though you may not be able to post a comment on lines outside of the current changes +/- 3 lines.

File: windows/security/threat-protection/microsoft-defender-atp/respond-file-alerts.md [hunk 1]
```diff
@@ -130,41 +130,41 @@ You can roll back and remove a file from quarantine if you’ve determined that
 
 > [!NOTE]
 > In some scenarios, the **ThreatName** may appear as: EUS:Win32/CustomEnterpriseBlock!cl.
-> 
+>
 > Defender for Endpoint will restore all custom blocked files that were quarantined on this device in the last 30 days.
 
 > [!Important]
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/respond-file-alerts.md:136]:
```suggestion
> [!IMPORTANT]
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - one minor additional edit for the sake of consistency.

**Comment by illfated** [comment]
I agree. I will merge the suggestion.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by illfated** [comment]
#sign-off

**Comment by illfated** [comment]
I made some mistakes while trying to clean up my branches. Yesterday was particularly difficult for me, I did not pay attention closely enough.

**Pull request merged**